packages/patterns-react/src/patterns/FeaturedLink/README.md

```diff
@@ -0,0 +1,108 @@
+# Featued Link Cards
+
+> The Featured Link Cards pattern is to be utilized within IBM.com.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+
+@import '@carbon/ibmdotcom-styles/scss/patterns/featuredlink/index.scss';
+```
+
+> 💡 Only import font's once per usage
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { CardArray } from '@carbon/ibmdotcom-patterns-react';
+import 'yourapplication.scss';
+
+const title = 'Lorem ipsum dolor sit amet.';
+
+const content = [
+  {
+    title: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit',
+    copy:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+    link: {
+      target: '_blank',
+      href: 'https://www.example.com',
+    },
+  },
+];
+
+function App() {
+  return <FeaturedLink title={title} content={content} />;
+}
+
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+> 💡 Don't forget to import the Featured Link styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+#### Feature Flags
+
+To utilize the following features, set the following variable's to `true` within
+your `.env` file or your application build settings.
+
+```
+DDS_FEATURED_LINK=true
```

Comment:
Should also include instructions to turn on `DDS_IMAGE_COMPONENT`

Reply:
@jeff updated